### PR TITLE
add support for range download

### PR DESCRIPTION
**Pull request by acasadoalonso**
This PR supports two new parameters on download:

&fromID=
&tillID=

In order to restrict the range of thevdownload entries

### CODE REVIEW

File: download/index.php [hunk 1]
```diff
@@ -16,6 +16,19 @@
     $filter[] = 'dev_id IN ('.$qm.')';
     $params = array_merge($params, $regs);
 }
+
+if (!empty($_GET['from_id'])) {
+    $fromid = $_GET['from_id'];
+    if (!empty($_GET['till_id'])) {
+	    $tillid = $_GET['till_id'];
+    } else {
+	    $tillid = 'FFFFFF';
```
Comment (snip) [inline download/index.php]: I'm not sure it is a good idea to have fixed maximum value for ID (the format of ID may change)
Why not generating an other filter in this case?

**Comment by acasadoalonso** [comment]
The end is only in the case that the user request fromID but not tillID
So we asked until the end 

Enviado desde mi iPhone.
AC/.

**Comment by snip** [comment]
> The end is only in the case that the user request fromID but not tillID So we asked until the end 

Yes but maybe tomorrow the last ID can be something else than FFFFFF.

**Comment by acasadoalonso** [comment]
That is true, but right now that is the end.
If some day we change that, we will change to the new end

AC/.

**Comment by snip** [comment]
My proposed change to your commit:
https://github.com/acasadoalonso/ogn-ddb/pull/1/files

Are you ok with that @acasadoalonso ?

**Comment by acasadoalonso** [comment]
Fully agree
Thx Seb

**Pull request merged**